data/man/vifm.1

```diff
@@ -4174,6 +4174,8 @@ since it cannot be selected
 .IP \- 2
 %z - short tips/tricks/hints that chosen randomly after one minute period
 .IP \- 2
+%{<expr>} - evaluate arbitrary vifm expression '<expr>', e.g. '&sort'
```

Comment:
`vifm/data/vim/doc/app/vifm-app.txt` also needs to be updated accordingly.

Reply:
Done.